package.json

```diff
@@ -41,7 +41,8 @@
     "typescript": "^4.9.4"
   },
   "dependencies": {
-    "ethers": "^5.7.2",
+    "@ambire/signature-validator": "^1.3.1",
+    "ethers": "5.7.2",
```

Comment:
Is ok to use ethers v5 instead of v6?

Reply:
yes, only the signature validator requires ethers and it does not support ethers v6